backend/src/main/java/com/google/capstone/TextToSpeech.java

```diff
@@ -59,11 +60,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
   }
 
+  private boolean inStorage(String objectIdString, String textString) {
+    Blob blob = storage.get(BlobId.of(BUCKET_NAME, objectIdString));
+    if (blob != null) {
+      String transcript = blob.getMetadata().get("audioTranscript");
```

Comment:
More often people use a hash function to produce a [fingerprint](https://en.wikipedia.org/wiki/Fingerprint_(computing)) to save storage space.

Reply:
is there a hash function or an algorithm you recommend for generating this?

Reply:
Should I use String.hashcode()?

Reply:
Discussed offline.